code/game/objects/items/RCD.dm

```diff
@@ -945,37 +962,14 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 /obj/item/construction/rcd/arcd
 	name = "advanced rapid-construction-device (ARCD)"
 	desc = "A prototype RCD with ranged capability and extended capacity. Reload with iron, plasteel, glass or compressed matter cartridges."
```

Comment:
If the matter is infinite, saying you can recharge seems redundant.

Reply:
Good point I'll change that